vividus-plugin-visual/src/main/java/org/vividus/visual/storage/BaselineStorageServiceLocator.java

```diff
@@ -0,0 +1,53 @@
+package org.vividus.visual.storage;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.BeanCreationException;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+
+public class BaselineStorageServiceLocator implements ApplicationContextAware
```

Comment:
```suggestion
public class BaselineStorageServiceLocator implements BeanFactoryAware
```
`BeanFactoryAware` should be enough, and `spring-beans` dependency can be used instead of `spring-context`